src/System.Net.Http/tests/FunctionalTests/CustomContent.cs

```diff
@@ -0,0 +1,138 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using System.Text;
+
+namespace System.Net.Http.Functional.Tests
+{
+    public class CustomContent : StreamContent
```

Comment:
I see you like things `sealed`. :)

I've usually never added `sealed` to test code unless it is being put into Common tree where we want other people to re-use it.

But I can add `sealed` if u want.